### PR TITLE
Pretty :) at least a little more

### LINKED ISSUE #105: Event Stats Page Cleanup
Make it look good - @jeffaburke @JoeJanickiChSh are either of you interested in looking at this?

### PR DESCRIPTION
**Pull request by jeffaburke**
Well I did a lot more than the original issue but major changes
- Linting of all jinja files
- Updates the look of most tables (including rounding, change and header with links into separate buttons), they are also styled globally (no need to add classes)
- New theme (New colors, dark dropdowns)

Closes #105

### CODE REVIEW

File: signinapp/templates/search/base.html.jinja2 [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
     <div class="row">
       {{ render_messages() }}
       <div class="col">{{ render_form(form, extra_classes="search-form") }}</div>
       <div class="col-lg-6 py-3">
-        <table class="table table-striped table-hover table-dark">
-          {%- block searchtable -%}
-          {%- endblock -%}
-        </table>
+        <div class="table-responsive">
+          <table>
```
Comment (bot190) [inline signinapp/templates/search/base.html.jinja2:13]: Can we put this class on the table itself so we don't need the extra element or does that break stuff?

**Review by bot190 — COMMENTED** [review]
I still need to load this up and checkout out the changes, but wanted to get you thinking about automatically linting the jinja files as a PR check. We probably also want to add the linter to our requirements.txt (or add requirements-dev.txt)

File: signinapp/templates/search/base.html.jinja2 (hunk 1 above)
Comment (jeffaburke) [reply signinapp/templates/search/base.html.jinja2:13]: according to the bootstrap docs, the table has to be wrapped with `table-responsive` in order for it to work. The `table-responsive` class allows for phone users and smaller screens to scroll tables.

**Comment by jeffaburke** [comment]
> I still need to load this up and checkout out the changes, but wanted to get you thinking about automatically linting the jinja files as a PR check. We probably also want to add the linter to our requirements.txt (or add requirements-dev.txt)

I'm using djLint, I'll add it to the `requirements.txt`, but how do I let people know what VS Code extensions to install?

**Comment by msoucy** [comment]
You can add a file to the `.vscode` folder like [here](https://code.visualstudio.com/docs/editor/extension-marketplace#_workspace-recommended-extensions)

File: signinapp/templates/subteam.html.jinja2 [hunk 2]
```diff
@@ -1,19 +1,23 @@
...
-        </ul>
+  <div class="container">
+    <h1>{{ subteam.name }}</h1>
+    <div class="row">
+      <div class="col">
+        <div>
+          <ul>
+            {%- for member in subteam.members -%}
+              <li>
+                <a href="{{ url_for('user.profile', username=member.username)}}">{{ member.name }}</a>
```
Comment (bot190) [inline signinapp/templates/subteam.html.jinja2:15]: Not a blocker, but the default link styling is hard to see on the background. We should probably update the color used for links to something lighter.

**Review by bot190 — APPROVED** [review]

**Pull request merged**